docs/public-networks/concepts/node-clients.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Node clients
+sidebar_position: 1
+description: Learn about execution and consensus clients.
+tags:
+  - public networks
+---
+
+# Node clients
+
+Ethereum's Proof of Stake (PoS) protocol leverages two separate P2P networks supporting 
+separate clients. Execution clients gossip transactions over their network, enabling them to manage 
+their local transaction pool. Consensus clients gossip blocks over their network, enabling consensus 
+and chain growth. A validator node also runs the [validator client]((#validator-clients).
+
+:::info
+As as result of [the 2022 Merge](https://ethereum.org/en/upgrades/merge/), Ethereum Mainnet 
+transitioned from Proof of Work (PoW) to [PoS](proof-of-stake/index.md) consensus.
+:::
+
+## Execution and consensus clients
+
+Under PoS, a full Ethereum Mainnet node is a combination of an execution client (previously called 
+an [Eth1 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/) client) and a 
+consensus client (previously called an 
+[Eth2 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)). The consensus client 
+communicates with the execution client via the [Engine API](../how-to/use-engine-api.md).
```

Comment:
Was probably in the original text, but we generally try and avoid using "via".

Reply:
image also updated to rmv Engine API bi-directionality as per @jflo